### DISCUSSION TITLE
"Range Error: Applying a mismatched transaction" with AlpineJS v3

### DISCUSSION THREAD

**Discussion by jamilabreu** [alpinejs/alpine#1938] [1. Help]
Would love some voices in this discussion: https://github.com/ueberdosis/tiptap/issues/1515

Description
I got Range Error: Applying a mismatched transaction when clicking on a menu button with AlpineJS v3.

I created a CodeSandbox to help you debug the issue: https://codesandbox.io/s/tiptap-v2-alpinejs-v3-s5x2o

**Comment by KevinBatdorf** [comment]
I added another reply. Basically there's no reason to set up the editor like that as it doesn't need to update to changes.

**Reply by EasterPeanut** [reply]
Yes in most cases the editor does not need to be reactive, but what about menu buttons with active classes? I can't figure out how to do that if the editor is not in x-data. 

Edit: worked around that by reassigning Alpine states in Tiptap's event listeners.

**Comment by KevinBatdorf** [comment]
There was another approach discussed here as well, which will let you scope the library without it being reactive: https://github.com/alpinejs/alpine/discussions/1953#discussioncomment-1203157

There might be room to expand on this as a feature though within Alpine since multiple libraries seem to have issues.
